spec/memoized_spec.cr

```diff
@@ -10,6 +10,35 @@ describe Memoized do
     msg.get.should eq(cnt)
   end
 
+  it "can be invoked with a block" do
+    msg = Memoized(Int32).new do
+      Dir["/tmp/*"].size
+    end
+
+    cnt = msg.get
+    cnt.should be_a(Int32)
+  end
+
+  it "can be invoked with a timespan and a block" do
+    msg = Memoized(Int32).new(1.minute) do
+      Dir["/tmp/*"].size
+    end
+
+    cnt = msg.get
+    cnt.should be_a(Int32)
+    msg.get.should eq(cnt)
+  end
+
+  it "can be invoked with a timespan and a path" do
```

Comment:
Tiny typo. This should be `with a path and a block`, right? 😃

Reply:
Heh, yeah sorry bout that ;)